libs/util/shared/src/lib/utils/fuzzy-filter.ts

```diff
@@ -0,0 +1,32 @@
+type FuzzyFilter = (input: string) => boolean
+
+function normalize(input: string) {
+  return input
+    .toLowerCase()
+    .normalize('NFD') // explode composite chars (e.g. é) into multiple chars
+    .replace(/[\u0300-\u036f]/g, '') // remove accents
+    .replace(/œ/g, 'oe') // remove accents
```

Comment:
Looking at french specificities, `æ` is not relevant in modern french anymore, is it?

Reply:
not ae but oe yes, like oeuf :)

Reply:
yep, I just saw that æ also ([used to](https://fr.wikipedia.org/wiki/%C3%86)) exist, but seems to have been replaced mostly by "accent aigu" nowadays, if I understand right :-)

Reply:
To be honest I would have never thought of it, but I encountered a metadata record with it!